wiki/People/Tournament_Committee/en.md

```diff
@@ -26,7 +33,111 @@ Their responsibilities include:
+
+### Public Communication
+
+Public communication by the Tournament Committee is handled through several channels. The purposes for these sometimes overlap, but all are official sources of information kept up-to-date by the Tournament Committee.
+
+The Tournament Committee maintains an [account on X](https://twitter.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
+
+The [Community Tournament Status Tracker](https://docs.google.com/spreadsheets/d/1bV5MyrJZI0F52Bx9EcYxdfRu8qGnhWRBKdXHK9uPbO0/edit?usp=sharing) is a publicly viewable spreadsheet detailing the status of all tournaments that have requested official support, i.e. all tournaments that requested screening. This sheet shows all tournaments dating back to January 14th, 2023, and contains a description of the status of the tournament at any given time, including whether a badge was applied or denied. If a badge was denied, the reason for denial is stated on the sheet.
+
+The [Official Tournament Support Updates thread](/community/forums/topics/1715676) is a pinned thread in the [Tournaments & Contests forum](/community/forums/126) which contains all updates to the Official Tournament Support wiki dating back to February 1st, 2023. This thread is updated and bumped every time a change is made to the Official Tournament Support wiki. This thread also contains links to the Official Support wiki and the Community Tournament Status Tracker.
+
+### Adding New Members
+
+New members to the Tournament Committee must meet the following basic criteria before being considered for the position at all:
+- Be at least 18 years old
+- Pass a security check
+
+Whether a prospective member applies via email or is brought forward by an existing member, the process is the same. A thread is either automatically created by the email application or created manually by a member of the Tournament Committee, and the whole committee is notified. The candidate is discussed in-depth, with special consideration paid to the uniqueness of the candidate compared to current members and the candidate’s history in the tournament scene. Assuming discussion is mixed or mostly positive, an anonymous vote is held to determine whether the candidate is added.
```

Comment:
```suggestion
Whether a prospective member applies via email or is brought forward by an existing member, the process is the same. The Tournament Committee holds an in-depth discussion on the candidate. Special attention is paid towards their activity in the tournament scene, stature, community contributions, and how they would benefit the Committee. Following said discussion, an anonymous vote is held to determine whether to add the candidate to the Committee.
```

Reply:
Added except lowercased committee when alone, and changed towards to toward, as towards is not a word.

Reply:
Nevermind apparently it's a word in British English. I hate dialects.

Reply:
TIL